airbyte-integrations/connectors/source-fullstory/source_fullstory/manifest.yaml

```diff
@@ -0,0 +1,188 @@
+version: "0.29.0"
+
+definitions:
+  requester:
+    url_base: "https://api.fullstory.com"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+
+  large_value_filter:
+    type: CustomFilter
+    filter_function: |
+      def filter_records(records, stream_state, config):
+          max_state_value = stream_state.get("createdAt", "9999-04-12T18:13:36.000Z")
+          if max_state_value == "9999-04-12T18:13:36.000Z":
+              return []
+          return records
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      type: RecordSelector
+      extractor:
+        type: DpathExtractor
+        field_path: ["{{ parameters.name }}"]
+      paginator:
+        type: NoPagination
+      # paginator:
+      #   type: "DefaultPaginator"
+      #   page_size_option:
+      #     type: "RequestOption"
+      #     inject_into: "request_parameter"
+      #     field_name: "{{ response['meta']['per_page'] }}"
+      #   pagination_strategy:
+      #     type: "PageIncrement"
+      #     page_size: 5
+      # page_token_option:
+      #   type: "RequestOption"
+      #   inject_into: "request_parameter"
+      #   field_name: "{{ response['meta']['page'] }}"
+
+  operations_stream:
+    type: DeclarativeStream
+    $parameters:
+      name: "operations"
+      path: "/operations/v1"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+      filters:
+        - $ref: "#/definitions/large_value_filter"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: "createdAt"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+      cursor_granularity: "PT0.000001S"
+      lookback_window: "P31D"
+      start_datetime:
+        datetime: "{{ config['start_date'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+      end_datetime:
+        datetime: "{{ today_utc() }}"
+        datetime_format: "%Y-%m-%d"
+      step: "P1M"
+
+  sessions_stream:
+    type: DeclarativeStream
+    $parameters:
+      name: "sessions"
+      path: "/sessions/v2?{{ 'uid=' ~ config['uid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+
+  segments_stream:
+    type: DeclarativeStream
+    $parameters:
+      name: "segments"
+      path: "/segments/v1"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        filters:
+          - $ref: "#/definitions/large_value_filter"
```

Comment:
https://developer.fullstory.com/server/v1/segments/list-segments/
This stream supports next token pagination, can you implement this?

Reply:
yeah

Reply:
implemented

> Waiting contributor update the code. The version right now is not possible to reproduce issue

@marcosmarxm, it was actually due to a formatting error while implementing the base_stream, it's resolved now, thanks. have also updated the connector with the requested changes.